tests/test_option_model.py

```diff
@@ -18,32 +26,46 @@ def test_default_option_model():
     obj4 = type2("obj4")
     obj9 = type2("obj9")
 
-    def _simulate(state, action):
-        next_state = state.copy()
-        obj = list(state)[0]
-        next_state.set(obj, "feat3",
-                       next_state.get(obj, "feat3") + action.arr[1])
-        return next_state
+    class _MockEnv:
+
+        @staticmethod
+        def simulate(state: State, action: Action) -> State:
+            """A mock simulate method."""
+            next_state = state.copy()
+            obj = list(state)[0]
+            next_state.set(obj, "feat3",
+                           next_state.get(obj, "feat3") + action.arr[1])
+            return next_state
 
-    params_space = Box(-10, 10, (2, ))
+        @property
+        def options(self) -> Set[ParameterizedOption]:
```

Comment:
we don't typically do mypy stuff in tests, and you don't do it on policy/initiable/etc below, so might as well remove here and in simulate() too

Reply:
(and in test_planning.py too)